Formula/node_exporter.rb

```diff
@@ -0,0 +1,65 @@
+class NodeExporter < Formula
+  desc "Prometheus exporter for machine metrics"
+  homepage "https://prometheus.io/"
+  url "https://github.com/prometheus/node_exporter/archive/v0.15.2.tar.gz"
+  sha256 "940b850376f94580e88b5e99926d92899975d5792d05709cbd9c48dab0a848ad"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/prometheus"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/node_exporter"
+
+    system "make", "build"
+    bin.install %w[node_exporter]
+    # libexec.install %w[consoles console_libraries]
```

Comment:
Why is this commented out?

Reply:
Copy/paste leftover from the `prometheus` formula.